# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qemu/zoom_to_fit.patch
# Copyright (C) 2020 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---


--- qemu-11.1.0/ui/gtk.c.vanilla	2026-08-18 17:22:26.751849643 +0200
+++ qemu-11.1.0/ui/gtk.c	2026-08-18 17:22:32.346971499 +0200
@@ -2490,7 +2490,7 @@
     GtkWidget *view_menu;
     GtkWidget *separator;
     QemuConsole *con;
-    bool zoom_to_fit = false;
+    bool zoom_to_fit = true;
     int vc, i;
 
     view_menu = gtk_menu_new();
